assets/styles/style.css

```diff
@@ -706,3 +723,149 @@ footer {
 }
 
 /* ====== CONTACT FORM END ==== */
+
+/* ====== CMODAL POPUP START ==== */
+.closeModal {
+  display: none;
+}
+
+.showModal {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  padding: 2em 1em;
+  background: #97949371;
+  backdrop-filter: blur(14px);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: var(--white);
+  padding: 1.4em 1em;
+  color: var(--black);
+}
```

Comment:
- [ ] You did an excellent job with your popup on the mobile version! However, there seems **to be a styling issue when viewing it on the smallest devices,** as the popup gets cut off. Would you mind adjusting the styling using percentages to ensure it displays properly? 😀

   ### Your design:
   ![image](https://github.com/Errytagedesign/microverse-portfolio/assets/107952724/eace7642-4cc3-4009-a8a0-bc85a870f4c3)